[2023-09-09T05:31:46.108+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-09T05:31:46.109+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table'
[2023-09-09T05:31:46.153+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-09T05:31:46.155+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table'
[2023-09-09T05:31:46.192+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-09T05:29:51.546808+00:00 [queued]>
[2023-09-09T05:31:46.208+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-09T05:29:51.546808+00:00 [queued]>
[2023-09-09T05:31:46.209+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-09-09T05:31:46.241+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryCreateEmptyTableOperator): create_listings_raw_table> on 2023-09-09 05:29:51.546808+00:00
[2023-09-09T05:31:46.252+0000] {standard_task_runner.py:57} INFO - Started process 3767 to run task
[2023-09-09T05:31:46.256+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Autotrader_ETL_DAG', 'create_listings_raw_table', 'manual__2023-09-09T05:29:51.546808+00:00', '--job-id', '504', '--raw', '--subdir', 'DAGS_FOLDER/autotrader_etl.py', '--cfg-path', '/tmp/tmpiye5hmhb']
[2023-09-09T05:31:46.264+0000] {standard_task_runner.py:85} INFO - Job 504: Subtask create_listings_raw_table
[2023-09-09T05:31:46.339+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-09T05:31:46.340+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-09T05:29:51.546808+00:00/task_id=create_listings_raw_table'
[2023-09-09T05:31:46.344+0000] {task_command.py:415} INFO - Running <TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-09T05:29:51.546808+00:00 [running]> on host 412bb32711f9
[2023-09-09T05:31:46.549+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Autotrader_ETL_DAG' AIRFLOW_CTX_TASK_ID='create_listings_raw_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-09T05:29:51.546808+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-09T05:29:51.546808+00:00'
[2023-09-09T05:31:46.576+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-09-09T05:31:46.586+0000] {base.py:73} INFO - Using connection ID 'google_cloud' for task execution.
[2023-09-09T05:31:46.587+0000] {bigquery.py:1507} INFO - Creating table
[2023-09-09T05:31:47.805+0000] {bigquery.py:1535} INFO - Table autotrader_staging.listings_raw already exists.
[2023-09-09T05:31:47.852+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Autotrader_ETL_DAG, task_id=create_listings_raw_table, execution_date=20230909T052951, start_date=20230909T053146, end_date=20230909T053147
[2023-09-09T05:31:47.920+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-09T05:31:47.981+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
